src/OrchardCore.Modules/OrchardCore.Setup/Startup.cs

```diff
@@ -23,7 +23,12 @@ public Startup(IShellConfiguration shellConfiguration)
             var configurationSection = shellConfiguration.GetSection("OrchardCore.Setup");
 
             _defaultCulture = configurationSection["DefaultCulture"] ?? "";
-            _supportedCultures = configurationSection.GetSection("SupportedCultures").Get<List<string>>().ToArray();
+            _supportedCultures = configurationSection.GetSection("SupportedCultures").Get<string[]>();
+            var results = configurationSection.GetSection("SupportedCultures").Get<List<string>>();
```

Comment:
Can I know what are you trying to do? If you want to add the `InvariantCulture` we can do it in the registration of the service

Reply:
Is it not what we do here?

Reply:
So what's you intend to do here

Reply:
We take the list of cultures in the appsettings.json file and we register those cultures in the service registration in the configure() method. I don't understand your comment. I explained yesterday why I was juggling with the `.Get<string[]>` and `.Get<List<string>>` that's because of the binding of the `.Get()` method. So I'm looking if someone wants to pass a string.empty in the SupportedCulture section of the appsetting.json file. If it's found then we add the string.empty value to the List<string> so that we support the InvariantCulture that is associated with it.

Reply:
.Get<string[]> will return an array containing { null, "fr", "en" } if we set a "" value in the appsettings.json file... that how the .Get() method binds data. We can't change that.

Reply:
https://github.com/aspnet/Configuration/issues/864

Reply:
Here an other one : https://github.com/aspnet/Configuration/issues/788

Reply:
Ok if I put { null, "fr", "en" } in the appsettings.json file it binds it to null when we do the `.Get<string[]>`  so it's not a possible workaround.